charts/enterprise-operator/values-openshift.yaml

```diff
@@ -119,60 +122,89 @@ relatedImages:
   - 107.0.1.8507-1_1.25.0
   - 107.0.1.8507-1_1.26.0
   - 107.0.1.8507-1_1.27.0
+  - 107.0.1.8507-1_1.28.0
+  - 107.0.10.8627-1
+  - 107.0.10.8627-1_1.25.0
+  - 107.0.10.8627-1_1.26.0
+  - 107.0.10.8627-1_1.27.0
+  - 107.0.10.8627-1_1.28.0
+  - 107.0.11.8645-1
+  - 107.0.11.8645-1_1.25.0
+  - 107.0.11.8645-1_1.26.0
+  - 107.0.11.8645-1_1.27.0
+  - 107.0.11.8645-1_1.28.0
   - 107.0.2.8531-1
   - 107.0.2.8531-1_1.25.0
   - 107.0.2.8531-1_1.26.0
   - 107.0.2.8531-1_1.27.0
+  - 107.0.2.8531-1_1.28.0
   - 107.0.3.8550-1
   - 107.0.3.8550-1_1.25.0
   - 107.0.3.8550-1_1.26.0
   - 107.0.3.8550-1_1.27.0
+  - 107.0.3.8550-1_1.28.0
   - 107.0.4.8567-1
   - 107.0.4.8567-1_1.25.0
   - 107.0.4.8567-1_1.26.0
   - 107.0.4.8567-1_1.27.0
+  - 107.0.4.8567-1_1.28.0
   - 107.0.6.8587-1
   - 107.0.6.8587-1_1.25.0
   - 107.0.6.8587-1_1.26.0
   - 107.0.6.8587-1_1.27.0
+  - 107.0.6.8587-1_1.28.0
   - 107.0.7.8596-1
   - 107.0.7.8596-1_1.25.0
   - 107.0.7.8596-1_1.26.0
   - 107.0.7.8596-1_1.27.0
+  - 107.0.7.8596-1_1.28.0
   - 107.0.8.8615-1
   - 107.0.8.8615-1_1.25.0
   - 107.0.8.8615-1_1.26.0
   - 107.0.8.8615-1_1.27.0
+  - 107.0.8.8615-1_1.28.0
   - 107.0.9.8621-1
   - 107.0.9.8621-1_1.25.0
   - 107.0.9.8621-1_1.26.0
   - 107.0.9.8621-1_1.27.0
+  - 107.0.9.8621-1_1.28.0
+  - 108.0.0.8694-1
+  - 108.0.0.8694-1_1.25.0
+  - 108.0.0.8694-1_1.26.0
+  - 108.0.0.8694-1_1.27.0
+  - 108.0.0.8694-1_1.28.0
   - 12.0.28.7763-1
   - 12.0.29.7785-1
   - 12.0.29.7785-1_1.25.0
   - 12.0.29.7785-1_1.26.0
   - 12.0.29.7785-1_1.27.0
+  - 12.0.29.7785-1_1.28.0
   - 12.0.30.7791-1
   - 12.0.30.7791-1_1.25.0
   - 12.0.30.7791-1_1.26.0
   - 12.0.30.7791-1_1.27.0
+  - 12.0.30.7791-1_1.28.0
   - 12.0.31.7825-1
   - 12.0.31.7825-1_1.25.0
   - 12.0.31.7825-1_1.26.0
   - 12.0.31.7825-1_1.27.0
+  - 12.0.31.7825-1_1.28.0
   - 12.0.32.7857-1
   - 12.0.32.7857-1_1.25.0
   - 12.0.32.7857-1_1.26.0
   - 12.0.32.7857-1_1.27.0
+  - 12.0.32.7857-1_1.28.0
   - 12.0.33.7866-1
   - 12.0.33.7866-1_1.25.0
   - 12.0.33.7866-1_1.26.0
   - 12.0.33.7866-1_1.27.0
+  - 12.0.33.7866-1_1.28.0
   - 13.10.0.8620-1
-  - 13.21.0.9059-1
```

Comment:
sanity check: 
- why did we get rid of 13.21.0.9059? 
  - I suppose because it got deep-archived, right?

Reply:
I think so, for the purposes of this PR, we don't have it in `release.json` anymore.